src/tests/ftest/pool/destroy.py

```diff
@@ -170,10 +171,10 @@ def test_destroy_single(self):
         :avocado: tags=pool_destroy_single
         """
         hostlist_servers = self.hostlist_servers[:1]
-        setid = self.params.get("setname", '/run/setnames/validsetname/')
+        set_id = self.params.get("setname", '/run/setnames/validsetname/')
```

Comment:
Unrelated, but I like to name the variable to match
```suggestion
        setname = self.params.get("setname", '/run/setnames/validsetname/')
```

Reply:
Makes sense